Expensify/App
# Refactor WorkspaceInitialPage to no longer use withFullPolicy

@arosiclair Please review

### Details
Refactor WorkspaceInitialPage to no longer use `withFullPolicy` and use the `FullPageNotFoundView` instead.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/218369

### Tests/QA
- Create a Workspace if you do not have one already
- Click on Settings -> [Workspace]
- Make sure you can interact with the page fully and without any errors:
https://user-images.githubusercontent.com/4741899/185338389-1b69cfd1-4bf9-424d-934d-06635a312134.mp4

---

- Using the URL bar above, change the workspace ID to something else.
- Make sure you see that the workspace is shown to be "Not found":
https://user-images.githubusercontent.com/4741899/185338667-4492d192-998d-4676-a362-42a92a39e2d4.mp4

---

- Using the JS console, switch the network tab to off
- Navigate to the workspace, make sure you are still able to and you load the page fully like before.

---

- [ ] For all of the above make sure there are no JS console errors while you do any of this.
